packages/field-plugin/src/messaging/pluginMessage/containerToPluginMessage/Asset.test.ts

```diff
@@ -35,24 +56,8 @@ describe('Asset', function () {
         assetFromAssetSelectedMessage(assetSelectedMessage),
       ).toHaveProperty('filename')
     })
-    it('keeps unknown properties', () => {
```

Comment:
This was intended to allow properties that might be added on the storyfront side, but now that we're wrapping all the "potential" additions under "asset" property, we no longer need this.